Files/UserControls/InnerNavigationToolbar.xaml

```diff
@@ -486,10 +486,12 @@
                                 </RadioButton>
 
                                 <TextBlock
+                                    x:Name="NavToolbarDetailsHeader"
                                     x:Uid="NavToolbarDetailsHeader"
                                     Grid.Row="0"
                                     Grid.Column="1"
                                     VerticalAlignment="Center"
+                                    PointerReleased="NavToolbarDetailsHeader_PointerReleased"
```

Comment:
Is there a click event?

Reply:
No, not on TextBlock.

Reply:
Will this work on touch screens?

Reply:
@yaichenbaum Probably not. I don't think this is necessary. This pr is just a practical addition but not essential. If we want it to also work on a tablet, we will have to make xaml more complex (hyperlink with a style, button template, ...)

Reply:
It's probably not too hard to restyle a button for this, I think it's important to allow support touch screens as the main button is a little small. I can work on this change if you want.

Reply:
You can do it if you want to. Here is my current schedule:
- Experimental ShowFolderSize: I have to test the clouds and ftp disks. The pr should arrive within 2 days.
- Search filter: Right after.
- This pr then, if it is not finished.

Reply:
I'll work on this 👍